app/controllers/councillor_contributions_controller.rb

```diff
@@ -9,8 +9,9 @@ def new
     else
       @councillor_contribution = CouncillorContribution.new
     end
-
-    @councillor_contribution.suggested_councillors.build({email: nil, name: nil})
+    if @councillor_contribution.suggested_councillors.empty? || @councillor_contribution.suggested_councillors.last.valid?
```

Comment:
This looks like it does the job to me 👍  I think we need an integration test to describe this behavior change.